Visual Studio/Framework_EGC.vcxproj

```diff
@@ -96,32 +96,32 @@
     <ProjectGuid>{FB43B467-42CC-458C-9556-597B025830F7}</ProjectGuid>
     <Keyword>Win32Proj</Keyword>
     <RootNamespace>Framework_EGC</RootNamespace>
-    <WindowsTargetPlatformVersion>10.0.17134.0</WindowsTargetPlatformVersion>
+    <WindowsTargetPlatformVersion>$(SDKVersion)</WindowsTargetPlatformVersion>
```

Comment:
Din pacate la mine (Visual Studio 2017) detecteaza incorect ca SDK instalat 8.1 si nu cel de Windows 10 instalat (10.0.17763.0). Se rezolva tot prin re-target si apoi se compileaza. Posibil ca pe Visual Studio 2019 sa mearga ok setarea variabilei SDKVersion insa nu am testat. Anyways, este mult mai bine asa cum ai schimbat.

Reply:
Exista o [serie de inconsistente](https://developercommunity.visualstudio.com/content/problem/140294/windowstargetplatformversion-makes-it-impossible-t.html) intre VS 2015/17/19, care impiedica o solutie unica. Momentan propun sa o pastram asa pana la terminarea semestrului — in branch-urile marcate cu `feature/` am eliminat cu totul folderul `Visual Studio`, dat fiind ca solutia generata de CMake ne scapa de aceste batai de cap.